samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+        <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
```

Comment:
No need, I removed them and seems to work like before. I think it was some left-over from the previous version.